### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by aphelionz**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -385,24 +385,32 @@ To discover resources via other parameters
...
     >>> auth = HydroShareAuthBasic(username='myusername', password='mypassword')
     >>> hs = HydroShare(auth=auth)
-    
+
     >>> # Discover via creator, group, user, owner
-    >>> resources = hs.resources(creator="email or name")
-    >>> resources = hs.resources(user="email or name")
-    >>> resources = hs.resources(owner="email or name")
-    >>> resources = hs.resources(author="email or name")
+    >>> resources = hs.resources(creator="username")
```
Comment (sblack-usu) [inline docs/index.rst]: email should be working.

Comment (aphelionz) [reply docs/index.rst]: For all of them or just creator?

File: docs/index.rst [hunk 2]
```diff
@@ -385,24 +385,32 @@ To discover resources via other parameters
...
+    >>> resources = hs.resources(user="username")
+    >>> resources = hs.resources(owner="username")
+    >>> resources = hs.resources(author="username")
     >>> resources = hs.resources(group="id or name")
 
+**from_date** allows you to specify the earliest creation date to query for
```
Comment (sblack-usu) [inline docs/index.rst]: I think these dates take specific formats... maybe an example here would nice?

File: docs/index.rst (hunk 1 above)
Comment (sblack-usu) [reply docs/index.rst]: all of them

File: docs/index.rst (hunk 2 above)
Comment (aphelionz) [reply docs/index.rst]: I thought they just took datetime objects? Either way I'm happy to include an example

Comment (sblack-usu) [reply docs/index.rst]: Looks like you're correct

**Pull request merged**